optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -155,6 +174,34 @@ class AddBlurryImagePlaceholders {
     });
     return false;
   }
+
+  /**
+   * Checks if an image should have a blurred image placeholder.
+   * The current criteria for determining if a blurry image placeholder should
+   * be appeneded is that it should be a JPEG that is either an amp-img that is
+   * a responsive or a poster for an amp-video. These two use cases were the
+   * most found to be the most common places where a blurry image placeholder
+   * would likely want to be used through manual testing.
+   * @param {Node} node The DOM element that is being checked to see if it
+   * should have a blurred placeholder.
+   * @param {string} src The image source that is being checked.
+   * @param {string} tagName The type of element that is being checked.
+   * @return {boolean} Whether or not the element should have a blurred
+   * placeholder child.
+   * @private
+   */
+  shouldAddBlurryPlaceholder_(node, src, tagName) {
+    // Checks to see if the image is a jpeg.
+    if (!src.endsWith('.jpg') && !src.endsWith('jpeg')) {
+      return false;
+    }
+
```

Comment:
I remember we discussed this in the last PR. Can you add another check: 

```
if (isImage && attribs.noloading) {
   return false;
}
```

Or am I missing something?